From f48607b17d92fd08e72f3613dd13cfd3d45ff7d2 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Thu, 18 Sep 2025 11:40:14 +0200
Subject: [PATCH] arm64: dts: st: rename reserved memory device tree files

STM32MP2x board device trees are agnostic to the flavor, whether ca35tdcid
or cm33tdcid.

Rename reserved memory device tree files by removing the '-ca35tdcid'
suffix and fix include in corresponding board device tree files, for all
MP2x boards.

Change-Id: I889fbbbff7cf0ee5c882c0150b9cf69af570067d
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/491049
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 ...215f-dk-ca35tdcid-resmem.dtsi => stm32mp215f-dk-resmem.dtsi} | 0
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts                       | 2 +-
 ...235f-dk-ca35tdcid-resmem.dtsi => stm32mp235f-dk-resmem.dtsi} | 0
 arch/arm64/boot/dts/st/stm32mp235f-dk.dts                       | 2 +-
 ...257f-dk-ca35tdcid-resmem.dtsi => stm32mp257f-dk-resmem.dtsi} | 0
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts                       | 2 +-
 ...7f-ev1-ca35tdcid-resmem.dtsi => stm32mp257f-ev1-resmem.dtsi} | 0
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts                      | 2 +-
 8 files changed, 4 insertions(+), 4 deletions(-)
 rename arch/arm64/boot/dts/st/{stm32mp215f-dk-ca35tdcid-resmem.dtsi => stm32mp215f-dk-resmem.dtsi} (100%)
 rename arch/arm64/boot/dts/st/{stm32mp235f-dk-ca35tdcid-resmem.dtsi => stm32mp235f-dk-resmem.dtsi} (100%)
 rename arch/arm64/boot/dts/st/{stm32mp257f-dk-ca35tdcid-resmem.dtsi => stm32mp257f-dk-resmem.dtsi} (100%)
 rename arch/arm64/boot/dts/st/{stm32mp257f-ev1-ca35tdcid-resmem.dtsi => stm32mp257f-ev1-resmem.dtsi} (100%)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk-ca35tdcid-resmem.dtsi b/arch/arm64/boot/dts/st/stm32mp215f-dk-resmem.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/st/stm32mp215f-dk-ca35tdcid-resmem.dtsi
rename to arch/arm64/boot/dts/st/stm32mp215f-dk-resmem.dtsi
diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index 4771d4e7849f..7b5a2acb66c8 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -14,7 +14,7 @@
 #include "stm32mp21xf.dtsi"
 #include "stm32mp21-pinctrl.dtsi"
 #include "stm32mp21xxan-pinctrl.dtsi"
-#include "stm32mp215f-dk-ca35tdcid-resmem.dtsi"
+#include "stm32mp215f-dk-resmem.dtsi"
 
 / {
 	model = "STMicroelectronics STM32MP215F-DK Discovery Board";
diff --git a/arch/arm64/boot/dts/st/stm32mp235f-dk-ca35tdcid-resmem.dtsi b/arch/arm64/boot/dts/st/stm32mp235f-dk-resmem.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/st/stm32mp235f-dk-ca35tdcid-resmem.dtsi
rename to arch/arm64/boot/dts/st/stm32mp235f-dk-resmem.dtsi
diff --git a/arch/arm64/boot/dts/st/stm32mp235f-dk.dts b/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
index afcb847158cc..34b29d1675cc 100644
--- a/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
@@ -14,7 +14,7 @@
 #include "stm32mp23xf.dtsi"
 #include "stm32mp25-pinctrl.dtsi"
 #include "stm32mp25xxak-pinctrl.dtsi"
-#include "stm32mp235f-dk-ca35tdcid-resmem.dtsi"
+#include "stm32mp235f-dk-resmem.dtsi"
 
 / {
 	model = "STMicroelectronics STM32MP235F-DK Discovery Board";
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk-ca35tdcid-resmem.dtsi b/arch/arm64/boot/dts/st/stm32mp257f-dk-resmem.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/st/stm32mp257f-dk-ca35tdcid-resmem.dtsi
rename to arch/arm64/boot/dts/st/stm32mp257f-dk-resmem.dtsi
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index da066d58c7e9..6b50916c1f77 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -14,7 +14,7 @@
 #include "stm32mp25xf.dtsi"
 #include "stm32mp25-pinctrl.dtsi"
 #include "stm32mp25xxak-pinctrl.dtsi"
-#include "stm32mp257f-dk-ca35tdcid-resmem.dtsi"
+#include "stm32mp257f-dk-resmem.dtsi"
 
 / {
 	model = "STMicroelectronics STM32MP257F-DK Discovery Board";
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi b/arch/arm64/boot/dts/st/stm32mp257f-ev1-resmem.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
rename to arch/arm64/boot/dts/st/stm32mp257f-ev1-resmem.dtsi
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 4b0445a77610..12d38fd4dac7 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -14,7 +14,7 @@
 #include "stm32mp25xf.dtsi"
 #include "stm32mp25-pinctrl.dtsi"
 #include "stm32mp25xxai-pinctrl.dtsi"
-#include "stm32mp257f-ev1-ca35tdcid-resmem.dtsi"
+#include "stm32mp257f-ev1-resmem.dtsi"
 
 / {
 	model = "STMicroelectronics STM32MP257F-EV1 Evaluation Board";
-- 
2.53.0

